data/devices/logitech-MX-Master-3s.device

```diff
@@ -0,0 +1,5 @@
+# Logitech MX Master 3S
+[Device]
+Name=Logitech MX Master 3S
+DeviceMatch=bluetooth:046d:b034;usb:046d:c548
```

Comment:
```suggestion
DeviceMatch=bluetooth:046d:b034
```
Please, remove the receiver ID `usb:046d:c548`. It is not specific to this device, so it does not fit in this device file. :)